src/cmd-diff

```diff
@@ -31,7 +31,7 @@ class DiffBuildTarget:
 class Differ:
     name: str
     description: str
-    needs_ostree: bool
+    needs_ostree: str
```

Comment:
This would be cleaner as [an enum](https://docs.python.org/3/library/enum.html).